### DISCUSSION TITLE
Endpoint for Story Archive

### DISCUSSION THREAD

**Discussion by senolatac** [sbertix/Swiftagram#256] [Q&A] [answered]
<img width="891" alt="Screen Shot 2021-09-15 at 10 20 14" src="https://user-images.githubusercontent.com/27292083/133389009-7506bf3d-c73f-4223-97ad-5a07174eeaaa.png">

Which endpoint does provide this(details of archived story item) information? Do you have any suggestion?

**Comment by senolatac** [comment]
Tried this one: `/v1/feed/reels_media/?user_ids=highlight:${storyId}` but this endpoint always return 0 as viewer-count.

**Comment by sbertix** [comment, marked as answer]
Hey @senolatac, 

**Swiftagram** implementation focuses on personal accounts alone, as to bridge the gap with the previously available official libraries: you should not try to mimic the behaviour of the [_Instagram Graph API_](https://developers.facebook.com/docs/instagram-api), available for Business and Creator accounts. 

So this is, unfortunately, not a feature, and I don't see many chances of it becoming one (for the reason I mentioned above).
